aptos-move/e2e-tests/src/executor.rs

```diff
@@ -1133,7 +1133,7 @@ impl FakeExecutor {
             false,
         )
         .unwrap();
-        let mut session = vm.new_session(state_view, SessionId::void());
+        let mut session = vm.new_session(state_view, SessionId::void(), None);
```

Comment:
Sorry I don't have any context for this PR. If we don't pass None here, how should we construct a transaction_context here?

Reply:
Context:
- native function that access user txn parameters such as max gas, entry fn name, etc. are added
- passing None here means these functions abort
- this function is used for compiler testing, so successful execution of such a native on mainnet does not hold anymore.

We can probably create it and pass it from the compiler testing tool here. cc'ed you so that you are aware of this when this lands and is actually used on-chain

Reply:
Thanks for the info @georgemitenkov! Let's keep it None in this PR. I will take a deeper look later.